ros/src/computing/planning/motion/packages/astar_planner/nodes/velocity_set/velocity_set.cpp

```diff
@@ -370,12 +377,12 @@ EControl pointsDetection(const pcl::PointCloud<pcl::PointXYZ>& points, const int
                          const autoware_msgs::lane& lane, const CrossWalk& crosswalk, const VelocitySetInfo& vs_info,
                          int* obstacle_waypoint, ObstaclePoints* obstacle_points)
 {
-  if (points.empty() == true || closest_waypoint < 0)
+  if (/*points.empty() == true ||*/ closest_waypoint < 0)
```

Comment:
Please put curly braces around all if statements. While this is technically ok to do, it's terrible coding practice and makes for unsustainable code.

Reply:
I fixed here to check external input from decision maker.